cmd/adduser.go

```diff
@@ -133,6 +142,27 @@ func (p *AddUserParams) PreInteractive(ctx ActionCtx) error {
 		return err
 	}
 
+	ok, err := cli.PromptYN("Set response permissions?")
+	if err != nil {
+		return err
+	}
+	if ok {
+		p.respMax, err = cli.Prompt("Number of max responses", p.respMax, true, func(v string) error {
+			_, err := strconv.ParseInt(v, 10, 32)
+			if err != nil {
+				return err
+			}
+			return nil
+		})
+		p.respTTL, err = cli.Prompt("Response TTL duration", p.respTTL, true, func(v string) error {
```

Comment:
It is the response permission ttl or expiration. Not sure if we want to reword here.

Reply:
The field simply says how long the server will wait for the client to respond before removing the permission right? If that is the case, I wanted to avoid "Expiration" as we already use that name somewhere else, but really can be anything you like.

Reply:
Yes maybe say Response Permission TTL?